cmd/job-executor-service-lint/main.go

```diff
@@ -1,15 +1,22 @@
 package main
 
 import (
+	"flag"
 	"io/ioutil"
 	"keptn-contrib/job-executor-service/pkg/config"
 	"keptn-contrib/job-executor-service/pkg/utils"
 	"log"
-	"os"
 )
 
 func main() {
-	args := os.Args[1:]
+
+	// Parse the allowPrivilegedJobs flag that can be changed to match the behavior of the job-executor-service
```

Comment:
For completeness we should add info about this flag in the README, up to you if you wanna put it in this PR or create a separate issue for docs